mappings/net/minecraft/test/GameTest.mapping

```diff
@@ -48,3 +55,16 @@ CLASS net/minecraft/class_4517 net/minecraft/test/GameTest
 	METHOD method_32241 isFlaky ()Z
 	METHOD method_32242 getMaxAttempts ()I
 	METHOD method_32243 getRequiredSuccesses ()I
+	METHOD method_33315 tickTests ()V
+	METHOD method_33316 (Lnet/minecraft/class_4518;)V
+		ARG 1 listener
+	METHOD method_36059 getStructureBlockBlockEntity ()Lnet/minecraft/class_2633;
+	METHOD method_36060 runAtTick (JLjava/lang/Runnable;)V
+		ARG 1 tick
+		ARG 3 runnable
+	METHOD method_36061 getSize ()Lnet/minecraft/class_2382;
+	METHOD method_36062 getBoundingBox ()Lnet/minecraft/class_238;
+	METHOD method_36063 getElapsedMilliseconds ()J
```

Comment:
Just fyi, we usually call milliseconds like "time" and ticks "ticks", so you can use `getElapsedTime` if you feel `getElapsedMilliseconds` is too long. No need to change though